include/mpm.h

```diff
@@ -51,6 +51,9 @@ class MPM {
   //! Apply nodal tractions
   virtual bool apply_nodal_tractions() = 0;
 
+  //! Apply entity sets properties (e.g: node_sets or particle_sets)
```

Comment:
This looks like purely for particle_sets assigning material. So please rephrase.
```
//! Assign material properties to particle sets
virtual bool assign_material_particle_sets() = 0;
```

Reply:
We think that this function can be used to assign other properties to nodes sets or particles sets in the future, such as velocity constraints, tractions, so we use this name, and I can add an argument to decide which property is used if you think it's necessary.